Pages/funfacts.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+  <title>Fun Facts</title>
+  <link href= "styles.css" rel= "stylesheet"/>
+</head>
+<body>
+
+  <header>
+    <div class= "header-inline">
+      <img src= "https://www.deltavcodeschool.com/wp-content/uploads/deltav-logo.jpg"/>
+      <nav class= "primary-nav">
+        <ul>
+          <li><a href= "index.html">Home</a></li>
+          <li><a href= "eduandwork.html">Education and Work Experience</a></li>
+          <li><a href= "funfacts.html">Fun Facts</a></li>
+          <li><a href= "gamepage.html">Guessing Game</a></li>
+        </ul>
+      </nav>
+    </div>
+    <nav class= "social-nav">
+      <ul>
+        <li><a href= "https://www.facebook.com/dustin.yelland">Facebook</a></li>
+        <li><a href= "https://twitter.com/dustinoid2000">Twitter</a></li>
+        <li><a href= "https://github.com/dustinyschild">Github</a></li>
+      </ul>
+  </header>
+
+
+<h1>Fun Facts!</h1>
+<ul>
+  <li>I love running and playing ultimate frisbee</li>
+  <li>I have a total of 6 brothers and sisters</li>
+  <li>Although I do not consider myself a morning person, very rarely do I let myself sleep in</li>
+    <ul>
+      <li>Makena</li>
+      <li>Mara</li>
+      <li>Braden</li>
+      <li>Clayton</li>
+      <li>Ethan</li>
+      <li>Myla</li>
+    </ul>
```

Comment:
Nice idea on using a nested list! Would it make more sense for this list to follow the bullet about your brothers and sisters?

For future reference, nested lists should be placed inside the corresponding `<li>`, e.g.

```html
<ul>
  <li>Simple item</li>
  <li>
    Item with sub-points
    <ul>
      <li>Sub-point 1</li>
      <li>Sub-point 2</li>
    </ul>
  </li>
  <li>Something else</li>
</ul>
```